inspirehep/modules/fixtures/users.py

```diff
@@ -176,3 +177,29 @@ def init_users_and_permissions():
     init_roles()
     init_users()
     init_permissions()
+
+
+def init_authentication_token():
+    with db.session.begin_nested():
+        c = Client(
+            name='admin',
+            user_id=1,
+            is_internal=True,
+            is_confidential=False,
+            _default_scopes=""
+        )
+        c.gen_salt()
```

Comment:
do we need the salt?

Reply:
yes, it generates the client id which we use when creating the token